app/views/layouts/blacklight/base.html.erb

```diff
@@ -16,6 +16,27 @@
     <%= stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= csrf_meta_tags %>
     <%= content_for(:head) %>
+
+<% if ENV['MATOMO_SITE_ID'] and ENV['MATOMO_URL'] %>
+    <script type="text/javascript">
+    var _paq = window._paq || [];
+    /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+    _paq.push(['trackPageView']);
+    _paq.push(['enableLinkTracking']);
+    (function () {
+        var u = "<%= ENV['MATOMO_URL'] %>";
+        _paq.push(['setTrackerUrl', u + 'matomo.php']);
+        _paq.push(['setSiteId', <%= ENV['MATOMO_SITE_ID'] %>]);
```

Comment:
Default to 16. See https://analytics.libraries.psu.edu/matomo/index.php?module=CoreHome&action=index&date=yesterday&period=day&idSite=16

I created a "Scholarpshere Dev" site in matomo.